src/go/cmd/token_builders.go

```diff
@@ -258,9 +258,7 @@ func makeConstTokens(name *string, c Const, list *[]Token) {
 	makeToken(nil, nil, "\t", whitespace, list)
 	makeToken(&n, nil, *name, typeName, list)
 	makeToken(nil, nil, " ", whitespace, list)
-	if c.Type == "*ast.BinaryExpr" {
-		makeToken(nil, nil, "", memberName, list)
-	} else {
+	if c.Type != "skip" {
```

Comment:
How does this change the output?

Reply:
If the const type is not "skip" then it will print whatever type is assigned for instance:
const <name> <type> = <value> 
vs if type is equal to skip:
const <name> = <value>

The previous method added a superfluous token.